custom_components/truenas/__init__.py

```diff
@@ -45,6 +46,11 @@
 ]
 TIMEOUT = 10
 
+ATTR_ENCRYPT = "Encrypted"
+ATTR_IS_DECRYPTED = "Is decrypted"
+ATTR_POOL_NAME = "Pool Name"
+ATTR_POOL_ID = "ID"
```

Comment:
Let's pull these into the [const file](https://github.com/sdwilsh/hass-truenas/blob/main/custom_components/truenas/const.py), and then import them :)